index.html

```diff
@@ -52,6 +52,183 @@ <h2 class="contact__title">LET'S CONNECT</h2>
             </li>
         </ul>
     </section>
+    <section class="work">
+        <ul class="card__container">
+            <li class="card">
+
+                <div class="card__imgbox">
+                    <img src="./images/project_screenshots/Lock.png" alt="Lock-app" class="card__img">
+                </div>
+
+                <div class="card__details">
+                    <h3 class="card__text">Lock API</h3>
+                    <div class="card__canopy">
```

Comment:
- [ ] Great work on your `works section` But it would be better to try and stick to the original design as much as you can, especially by using the correct images and designing the cards like in the original design. This will  ensure consistency in the design of your portfolio

![image](https://user-images.githubusercontent.com/98453979/235091262-fcef21bb-faab-4fbf-8d9f-0bb47e3afaa9.png)

### Checkout the original design

![image](https://user-images.githubusercontent.com/98453979/235091460-c2af8e40-5554-425f-bb29-79386d943244.png)